news/2019-09-27-mappers-guild-september-newspost.md

```diff
@@ -0,0 +1,146 @@
+
+If you're curious and want to know more about the Guild and its quest system, please visit [the official Mappers' Guild website](https://mappersguild.com)!
+
+There are a lot of completed quests, so here we've picked out the ones involving the most mapsets and mappers. You can find even more Featured Artist content by searching the [beatmap listing](https://osu.ppy.sh/beatmapsets?q=featured%20artist)!
+
+![](https://assets.ppy.sh/artists/23/header.jpg)
+
+For this quest, the mappers had to **create and rank at least 6 mapsets of songs by Cranky. Half of the mapsets had to include difficulties of a game mode different from the other half, and each mapset had to be hosted by a different user**.
+
+The **Cranky Hybrid Pack** quest was completed by **[Icekalt](https://osu.ppy.sh/users/5410645)**, **[Kalibe](https://osu.ppy.sh/users/3376777)**, **[Nao Tomori](https://osu.ppy.sh/users/5364763)**, **[Mao](https://osu.ppy.sh/users/2204515)**, **[Myxo](https://osu.ppy.sh/users/2202645)**, **[ScubDomino](https://osu.ppy.sh/users/8972308)**, **[xLolicore-](https://osu.ppy.sh/users/4525153)**, **[Nwolf](https://osu.ppy.sh/users/1910766)**, **[Ascendance](https://osu.ppy.sh/users/2931883)**, **[JBHyperion](https://osu.ppy.sh/users/4879508)**, **[Du5t](https://osu.ppy.sh/users/6053071)** and **[Jemzuu](https://osu.ppy.sh/users/7890134)**.
+
+- [Cranky - Conflict](https://osu.ppy.sh/beatmapsets/936126) by [Icekalt](https://osu.ppy.sh/users/5410645)
+- [Cranky - Amanojaku](https://osu.ppy.sh/beatmapsets/978694) by [Mao](https://osu.ppy.sh/users/2204515)
+- [J. Pachelbel - Canon (Cranky Remix)](https://osu.ppy.sh/beatmapsets/978674) by [Nao Tomori](https://osu.ppy.sh/users/5364763)
+- [L. V. Beethoven - Moonlight Sonata (Cranky Remix)](https://osu.ppy.sh/beatmapsets/978759) by [Nwolf](https://osu.ppy.sh/users/1910766)
+- [Cranky - EMPEROR](https://osu.ppy.sh/beatmapsets/979438) by [JBHyperion](https://osu.ppy.sh/users/4879508)
+- [Cranky - JULIAN](https://osu.ppy.sh/beatmapsets/1008310) by [ScubDomino](https://osu.ppy.sh/users/8972308)
+
+![](https://assets.ppy.sh/artists/37/header.jpg)
+
+For this quest, the mappers had to **create and rank at least 6 mapsets of songs by The Flashbulb, each hosted by a different user**.
+
+**The Flashbulb Standard Beatmap Pack** quest was completed by **[Pachiru](https://osu.ppy.sh/users/2850983)**, **[Gabe](https://osu.ppy.sh/users/654108)**, **[Zetera](https://osu.ppy.sh/users/587737)**, **[polka](https://osu.ppy.sh/users/4191702)**, **[greenhue](https://osu.ppy.sh/users/4967662)**, **[Irreversible](https://osu.ppy.sh/users/1287964)**, **[MBomb](https://osu.ppy.sh/users/3071175)**, **[Mismagius](https://osu.ppy.sh/users/19048)**, **[Leader](https://osu.ppy.sh/users/631530)**, **[fieryrage](https://osu.ppy.sh/users/3533958)** and **[Deif](https://osu.ppy.sh/users/318565)**.
```

Comment:
misleading to call it just "standard" when 1 of the 6 mapsets was osu!catch and 2 of the 10 members (MBomb, Deif) were in the quest to do that set.

Reply:
I named it after the name of the quest on the mappers' guild website, I can eventually change this to hybrid if you think it's better

Reply:
the one on the website is just called `The Flashbulb Beatmap Pack` with the taiko one being `The Flashbulb Taiko Beatmap Pack` 

so I think hybrid would be better, it already doesn't match the mpg website listing, but would be more descriptive